public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -215,9 +229,25 @@ export const AnomalyHeatmapChart = React.memo(
       return false;
     };
 
+    // Custom hook to refresh all of the heatmap data when running a historical task
```

Comment:
Will it cause heavy load if query too much? Maybe we can increase interval between refresh query for HC?

Reply:
I think it's a fair point. I'm ok to increase the timeout - how about 30s?

Reply:
Sounds good, maybe confirm with UX design to check if this is reasonable and should we add some tooltip

Reply:
Sure, confirming with UX.

Reply:
Confirmed 30s is ok - no extra notices/wording is necessary.